src/payment/order-summary/OrderSummary.jsx

```diff
@@ -4,12 +4,13 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { FormattedMessage } from '@edx/frontend-i18n';
 
-import { basketSelector } from '../data/selectors';
+import { orderSummarySelector } from '../data/selectors';
 import BulkOrderSummaryTable from './BulkOrderSummaryTable';
 import SummaryTable from './SummaryTable';
 import TotalTable from './TotalTable';
 import CouponForm from './CouponForm';
 import DiscountOffers from './DiscountOffers';
+import CurrencyDisclaimer from '../CurrencyDisclaimer';
```

Comment:
Nit: Could move CurrencyDisclaimer into the order-summary directory.

Reply:
I considered that, if we assume it's always going to go with the order summary?

Reply:
Seems like order summary is wrapping a few disparate things already, so I'd say "suuure!"  Easy to change later if it gets put somewhere else.

Reply:
Did the thing.